File: code/1015/1015.js
const input = require("fs").readFileSync("/dev/stdin", "utf8");
const values = input.split(/\n|\s/);

const x1 = parseFloat(values[0]);
const y1 = parseFloat(values[1]);
const x2 = parseFloat(values[2]);
const y2 = parseFloat(values[3]);

console.log(Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2)).toFixed(4));

File: code/1015/1015.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        string line1, line2;
        double x1, y1, x2, y2;
        line1 = Console.ReadLine();
        line2 = Console.ReadLine();
        string[] values1 = line1.Split(' ');
        string[] values2 = line2.Split(' ');
        double.TryParse(values1[0], out x1);
        double.TryParse(values1[1], out y1);
        double.TryParse(values2[0], out x2);
        double.TryParse(values2[1], out y2);

        Console.WriteLine("{0:0.0000}", CalculateDistanceBetweenTwoPoints(x1, y1, x2, y2));
    }

    static double CalculateDistanceBetweenTwoPoints(double x1, double y1, double x2, double y2)
    {
        return Math.Sqrt(Math.Pow((x2 - x1), 2) + Math.Pow((y2 - y1), 2));
    }
}